pkg/autodiscovery/listeners/snmp.go

```diff
@@ -261,6 +274,19 @@ func (l *SNMPListener) checkDevices() {
 	}
 }
 
+func (l *SNMPListener) collectMetrics(subnets []snmpSubnet) {
+	sender, senderErr := aggregator.GetSender(senderCheckID)
```

Comment:
Using default aggregator. Is that the best way to get a sender?

Reply:
Not sure, but seems to be the standard across Go checks at least

Reply:
in the cluster-agent we are using Prometheus for the metrics, we have already a set of metrics regarding te cluster-checks.

Reply:
Do you have an example of that? (sending metrics via Prometheus)